types/lib/paypal.d.ts

```diff
@@ -25,6 +25,7 @@ export type DirectConfig = {
     logoImageUrl?: string;
     headerImageUrl?: string;
   };
+  payPalComplete?: boolean;
 };
 
 export type PayPalConfig = BraintreeConfig | DirectConfig;
```

Comment:
I think to match and hint at the underlying logic, we might consider the following:

```
export type PayPalCompleteConfig = {
  payPalComplete?: boolean;
};

export type PayPalConfig = BraintreeConfig | DirectConfig | PayPalCompleteConfig;
```

Reply:
I think this pull request should be closed because I'm confused about which version of the library this pertains to. By default, the documentation that initially displays (`latest`) shows this option under the PayPal section, but if you switch the documentation to `4.10.3` this is not an option. When I applied this to the version that is delivered via [js.recurly.com/v4/recurly.js](https://js.recurly.com/v4/recurly.js), this introduces strange behaviors with PayPal complete, but when left out, things work as expected. This leads me to believe the version served by js.recurly.com is actually `4.10.3` and not `latest`. So perhaps this actually isn't an option on the current (actual latest) version of the platform?

Reply:
These changes have been applied. I created a new `PayPalCompleteConfig`. I took a best guess at what other properties would be needed.

> I think to match and hint at the underlying logic, we might consider the following:
> 
> ```
> export type PayPalCompleteConfig = {
>   payPalComplete?: boolean;
> };
> 
> export type PayPalConfig = BraintreeConfig | DirectConfig | PayPalCompleteConfig;
> ```